pkg/xds/topology/outbound.go

```diff
@@ -348,7 +349,7 @@ func fillIngressOutbounds(
 					}
 					// this is necessary for correct spiffe generation for dp when
 					// traffic is routed: egress -> ingress -> egress
-					if mesh.ZoneEgressEnabled() && service.ExternalService {
```

Comment:
why this was removed?

Reply:
because it was unnecessary: https://github.com/kumahq/kuma/pull/5070/files/b9819c1c5b33b3ea64f854053bf8a476cf744d7a#diff-03935b6847c0459458aef35c9f6e46a8abbdbac4ab80c2f16b87687558359628R337

Reply:
link is broken, so
![image](https://user-images.githubusercontent.com/11655498/192760727-c8d33d61-1d78-480e-b6b1-34835f9d71a8.png)

Reply:
Ok, I see it now, thanks.